schemas/com.snowplowanalytics.monitoring/sink_write_failed/jsonschema/1-0-0

```diff
@@ -0,0 +1,37 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for a write to a sink failing",
+	"self": {
+		"vendor": "com.snowplowanalytics.monitoring",
+		"name": "sink_write_failed",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+
+	"type": "object",
+
+	"properties": {
+
+		"sink": {
+			"type": "string"
+		},
+
+		"failureCount": {
+			"type": "integer"
+		},
+
+		"initialFailureTime": {
+			"type": "number"
+		},
+
+		"lastRetryPeriod": {
+			"type": "number"
```

Comment:
number or integer for these three numeric fields? The Redshift schema says int, which is int4, so would need to be constrained to these values: -2147483648 to +2147483647

[1] http://docs.aws.amazon.com/redshift/latest/dg/r_Numeric_types201.html

For example numeric constraints see: https://github.com/snowplow/iglu-central/blob/master/schemas/org.w3/PerformanceTiming/jsonschema/1-0-0#L13